backoffice/src/gateways/mappers/BeaconResponseMapper.ts

```diff
@@ -77,6 +85,70 @@ export class BeaconResponseMapper implements IBeaconResponseMapper {
     ];
   }
 
+  public mapAccountHolder(
+    accountHolder: IAccountHolderResponse
+  ): IAccountHolder {
+    return {
+      id: accountHolder.id,
+      fullName: accountHolder.attributes.fullName || "",
+      email: accountHolder.attributes.email || "",
+      telephoneNumber: accountHolder.attributes.telephoneNumber || "",
```

Comment:
Consider moving mapping logic server side?

Reply:
Yeah fair point, it probably should be, but at the moment we're re-suing the beacon workflow and could cause issues